apps/play/src/components/SampleListSection.tsx

```
@@ -0,0 +1,83 @@
+              </div>
+              <button
+                class='delete-button'
+                onclick={(e) => handleDelete(sample, e)}
+                title={`Delete ${sample.name}`}
+              >
+                <svg
+                  width='16'
+                  height='16'
+                  viewBox='0 0 24 24'
+                  fill='currentColor'
+                >
```

Biome (2.5.0)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)